Tests/Marketplace/Tests/test_data/azure_sentinel_search_response.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
better to avoid having test data mocks as files, but a generlize way to create a pack search response, so if tomorrow i want to add a specific test i dont have to create a lot of garbage files - it really ease up the process of writing tests and not bound us to test files

can we do that here?